docs/src/site/optics.md

```diff
@@ -6,5 +6,12 @@ section: "optics"
 
 # Optics
 
+## Convert between two types
```

Comment:
I would like to rework that diagram eventually. It confused me totally when I tried to learn about monocle, because I constantly thought it was talking about inheritance and it took me awhile to realize that it is talking about transformations (I am UML damaged to certain degree). Thus I never understood why my lenses did not compose.